wdl/transforms/new-base/src/main/scala/wdl/transforms/base/ast2wdlom/GenericAst.scala

```diff
@@ -31,6 +31,15 @@ trait GenericAstNode {
     allTerminals.foldLeft[Option[GenericTerminal]](None)(foldFunction)
   }
 
+  // We would actually like to get the extent that is covered by this AST. A representation
+  // like:  (startLine, startColumn, endLine, endColumn) would be efficient, while conveying
+  // all the information needed for downstream analysis phases. However, it turns out that
+  // getting accurate information out of Hermes is not that simple. For now, we just
+  // get the initial source line, which is -more or less- accurate.
```

Comment:
I tried unsuccessfully to use the `firstTerminal` method from `wdl.transforms.base.ast2wdlom.GenericAstNote`. In the workflow case, it returns the workflow name, not the `workflow` keyword. For example, in workflow `foo`:

```
workflow foo {
}
```

first terminal will be `foo`. Taking the column number from that returns 10, instead of 0.